### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by colefordays**
I moved some of the db init steps to the setup instructions and deleted periods

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,33 +17,31 @@ This is an app for keeping track of tasks, projects, and skills. Use tasks as bu
...
 ```bash
 export FLASK_APP=api
 export FLASK_ENV=development
-flask init-db
-flask run
 ```
 
 For Windows cmd, use set instead of export:
 
 ```bash
```
Comment (nickpale) [inline README.md:37]: Did you see if this has its own syntax highlighting?

File: README.md [hunk 2]
```diff
@@ -17,33 +17,31 @@ This is an app for keeping track of tasks, projects, and skills. Use tasks as bu
 
         `venv\Scripts\activate`
 
-4. Install dependencies.
+4. Install dependencies
     - `pip install -r requirements.txt`
 
+5. Initialize Database
```
Comment (nickpale) [inline README.md:23]: Nice 🏆

**Review by nickpale — COMMENTED** [review]
Just a couple of questions

File: README.md (hunk 1 above)
Comment (colefordays) [reply README.md:37]: it doesnt

**Review by nickpale — APPROVED** [review]
💯

**Pull request merged**